src/test/java/teammates/test/driver/AssertHelper.java

```diff
@@ -198,4 +199,16 @@ public static void assertSameContentIgnoreOrder(List<?> a, List<?> b) {
 
     }
 
+    /**
+     * Asserts that the two given arrays have the same content, in the same order.
+     * @param expected : Array of expected status messages
+     * @param actual : Array of actual status messages
+     */
+    public static void assertAllContained(String[] expected, String[] actual) {
```

Comment:
Don't reinvent the wheel, `AssertJUnit#assertEquals` exists. Notice how the naming is named better?